include/FadeButton.h

```diff
@@ -52,14 +55,16 @@ public slots:
 
 
 private:
-	QTime m_stateTimer;
+	QTime m_activateStateTimer;
+
```

Comment:
Is this just a renaming?

Reply:
Good catch. At one point I implemented it via 2 timers, so i changed the names to remove ambiguity. I then removed the second timer but never undid the renaming.